### PR TITLE
V3.7.0

### PR DESCRIPTION
**Pull request by kagg-design**

### CODE REVIEW

**Comment by kagg-design** [comment]
@e271828- Could you please review and merge?

**Review by e271828- — COMMENTED** [review]
Looks good, but need the other ent fields.

File: .tests/php/unit/HCaptchaTestCase.php [hunk 1]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'default' => 'js.hcaptcha.com',
+				'helper'  => 'Hostname hCaptcha api.js is loaded from.',
+			],
+			'asset_host'               => [
+				'label'   => 'Asset Host',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Asset host for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:588]:
```suggestion
				'helper'  => 'Asset host for hCaptcha api.js (Enterprise)',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 2]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+			],
+			'endpoint'                 => [
+				'label'   => 'Endpoint',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Endpoint for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:594]:
```suggestion
				'helper'  => 'Endpoint for hCaptcha api.js (Enterprise)',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 3]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+			],
+			'host'                     => [
+				'label'   => 'Host',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Host for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:600]:
```suggestion
				'helper'  => 'See Enterprise docs',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 4]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+			],
+			'image_host'               => [
+				'label'   => 'Image Host',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Host for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:606]:
```suggestion
				'helper'  => 'See Enterprise docs',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 5]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+			],
+			'report_api'               => [
+				'label'   => 'Report API',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Report API for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:612]:
```suggestion
				'helper'  => 'See Enterprise docs',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 6]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
...
+			],
+			'sentry'                   => [
+				'label'   => 'Sentry',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'helper'  => 'Sentry for hCaptcha api.js.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:618]:
```suggestion
				'helper'  => 'See Enterprise docs',
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 7]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
 					)
 				),
 			],
+			'api_host'                 => [
+				'label'   => 'API Host',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'default' => 'js.hcaptcha.com',
+				'helper'  => 'Hostname hCaptcha api.js is loaded from.',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:582]:
```suggestion
				'helper'  => 'Hostname hCaptcha api.js is loaded from. (See Enterprise docs)',
```

File: src/php/Settings/General.php [hunk 8]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
 					)
 				),
 			],
+			'api_host'             => [
+				'label'   => __( 'API Host', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'default' => Main::API_HOST,
+				'helper'  => __( 'Hostname hCaptcha api.js is loaded from.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:383]:
```suggestion
				'helper'  => __( 'See Enterprise docs' ),
```

File: src/php/Settings/General.php [hunk 9]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'asset_host'           => [
+				'label'   => __( 'Asset Host', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Asset host for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:389]:
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: src/php/Settings/General.php [hunk 10]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'endpoint'             => [
+				'label'   => __( 'Endpoint', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Endpoint for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:395]:
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: src/php/Settings/General.php [hunk 11]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'host'                 => [
+				'label'   => __( 'Host', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Host for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:401]:
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: src/php/Settings/General.php [hunk 12]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'image_host'           => [
+				'label'   => __( 'Image Host', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Host for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:407]:
```suggestion
				'helper'  => __( 'Host for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: src/php/Settings/General.php [hunk 13]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'report_api'           => [
+				'label'   => __( 'Report API', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Report API for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:413]:
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: readme.txt [hunk 14]
```diff
@@ -540,6 +540,12 @@ Instructions for popular native integrations are below:
 
 == Changelog ==
 
+= 3.7.0 =
+* Tested with WooCommerce 8.5.
+* Added Enterprise settings for API host and Endpoint.
```
Comment (e271828-) [inline readme.txt:545]:
```suggestion
* Added optional Enterprise settings.
```

File: src/php/Settings/General.php [hunk 15]
```diff
@@ -369,6 +375,49 @@ public function init_form_fields() {
...
+			],
+			'sentry'               => [
+				'label'   => __( 'Sentry', 'hcaptcha-for-forms-and-more' ),
+				'type'    => 'text',
+				'section' => self::SECTION_ENTERPRISE,
+				'helper'  => __( 'Sentry for hCaptcha api.js.', 'hcaptcha-for-forms-and-more' ),
```
Comment (e271828-) [inline src/php/Settings/General.php:419]:
```suggestion
				'helper'  => __( 'See Enterprise docs', 'hcaptcha-for-forms-and-more' ),
```

File: .tests/php/unit/HCaptchaTestCase.php [hunk 16]
```diff
@@ -574,6 +574,49 @@ protected function get_test_general_form_fields(): array {
 					)
 				),
 			],
+			'api_host'                 => [
+				'label'   => 'API Host',
+				'type'    => 'text',
+				'section' => 'enterprise',
+				'default' => 'js.hcaptcha.com',
+				'helper'  => 'Hostname hCaptcha api.js is loaded from. (See Enterprise docs)',
```
Comment (e271828-) [inline .tests/php/unit/HCaptchaTestCase.php:582]:
```suggestion
				'helper'  => 'See Enterprise docs',
```

**Comment by kagg-design** [comment]
@e271828- Thank you for the fixes. I have added URL encoding, and it should work now.

**Review by e271828- — APPROVED** [review]

**Pull request merged**